crates/re_types/src/datatypes/tensor_data_ext.rs

```diff
@@ -154,6 +165,52 @@ impl TensorData {
             TensorBuffer::F32(buf) => Some(TensorElement::F32(buf[offset])),
             TensorBuffer::F64(buf) => Some(TensorElement::F64(buf[offset])),
             TensorBuffer::Jpeg(_) => None, // Too expensive to unpack here.
+            TensorBuffer::Nv12(_) => {
+                {
+                    // Returns the U32 packed RGBA value of the pixel at index [y, x] if it is valid.
+                    let [y, x] = index else {
+                        return None;
+                    };
+                    if let Some(
+                        [TensorElement::U8(r), TensorElement::U8(g), TensorElement::U8(b)],
+                    ) = self.get_nv12_pixel(*x, *y)
+                    {
+                        let mut rgba = 0;
+                        rgba |= (r as u32) << 24;
+                        rgba |= (g as u32) << 16;
+                        rgba |= (b as u32) << 8;
+                        rgba |= 0xff;
+                        Some(TensorElement::U32(rgba))
+                    } else {
+                        None
+                    }
+                }
+            }
+        }
+    }
+
+    pub fn get_nv12_pixel(&self, x: u64, y: u64) -> Option<[TensorElement; 3]> {
```

Comment:
The reasoning here is that it fits better with the existing `get` method...
```rs
    pub fn get(&self, index: &[u64]) -> Option<TensorElement> {
```
 This way it can just be plugged in wherever the existing get method is used, like here:
 https://github.com/luxonis/depthai-viewer/blob/54e32c8869827644a9b5d2460663bee9cde8ad63/crates/re_data_ui/src/image.rs#L672C1-L672C1

Reply:
I can still change it if you want me to, just to give you more context ;)

Reply:
k makes sense. Let's leave the return then as is :)
(but I still wouldn't mind the doc string ;))

Reply:
Doc string added